docs/AboutUs.md

```diff
@@ -19,15 +19,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Meng Qiaoran
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Qiaoran.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/Qiaoran-M)]
+[[portfolio](team/Qiaoran.md)]
```

Comment:
Same for the PPP filename as mentioned [here](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#ppp-format-2)